### PR TITLE
added className, id, as props, also changed object literally access o…

### PR DESCRIPTION
**Pull request by nicola-spire**
…f objects ex: a['b'] to dot notation ex: a.b

### CODE REVIEW

File: library/source/layouts/box_layout.tsx [hunk 1]
```diff
@@ -22,6 +22,12 @@ export interface Properties {
    *  height equal to the height of its content.
    */
   height?: number | string;
+  
+  /** The unique id of the VboxLayout HTML element. */
```
Comment (kranar) [inline library/source/layouts/box_layout.tsx]: This is not a VboxLayout.

File: library/source/layouts/box_layout.tsx [hunk 2]
```diff
@@ -22,6 +22,12 @@ export interface Properties {
...
+  id?: string;
+
+  /** Specifies the CSS class of the VboxLayout HTML element. */
```
Comment (kranar) [inline library/source/layouts/box_layout.tsx]: Not a VboxLayout

File: library/source/layouts/hbox_layout.tsx [hunk 3]
```diff
@@ -14,6 +14,12 @@ export interface Properties {
    */
   height?: number | string;
 
+  /** The unique id of the VboxLayout HTML element. */
```
Comment (kranar) [inline library/source/layouts/hbox_layout.tsx]: HBoxLayout rather than VboxLayout.

File: library/source/layouts/hbox_layout.tsx [hunk 4]
```diff
@@ -14,6 +14,12 @@ export interface Properties {
...
+  id?: string;
+
+  /** Specifies the CSS class of the VboxLayout HTML element. */
```
Comment (kranar) [inline library/source/layouts/hbox_layout.tsx]: Same issue.

File: library/source/layouts/vbox_layout.tsx [hunk 5]
```diff
@@ -14,6 +14,12 @@ export interface Properties {
    */
   height?: number | string;
 
+  /** The unique id of the VboxLayout HTML element. */
```
Comment (kranar) [inline library/source/layouts/vbox_layout.tsx]: VBoxLayout instead of VboxLayout

File: library/source/layouts/vbox_layout.tsx [hunk 6]
```diff
@@ -14,6 +14,12 @@ export interface Properties {
...
+  id?: string;
+
+  /** Specifies the CSS class of the VboxLayout HTML element. */
```
Comment (kranar) [inline library/source/layouts/vbox_layout.tsx]: Same typo.

**Comment by nicola-spire** [comment]
Oh man, thank you!

**Pull request merged**